### ISSUE TITLE
ATO pane enhancements

### ISSUE THREAD

**Issue by ghost** [dcs-liberation/dcs_liberation#2027]
Currently and by default, the top left ATO flights pane is not sortable in any way and lacks some useful information.

The ATO pane of Liberation should offer sorting options, where 'TO Time' or 'TOT Time' sorting is the default sort order, ascending or descending.  Additional data on the package could include: number of flights, start type.

Lastly, re-sorting the ATO pane should happen after submission of new packages, during the auto-refresh of that pane.

**Additional context**
Currently, the TO/TOT times of the planned flights are unsorted, where a 0:40:00 TOT might appear below a 1:10:00 TOT.

**Comment by ghost** [comment]
This enhancement to the ATO pane could also include options for displaying spawn times and sorting by those, as a mechanism intended to help players de-conflict deck issues on carriers with the AI crowding and getting things gummed up.

Could be combined with a check in the 'Take Off' button logic that checks for TOTs, offering the player to automatically fix them.  In that logic, there could be sorting and re-TOT-ing to space flight spawns 3-5 mins apart. (New Enhancement?)

**Comment by HILOK** [comment]
yes, totally needed! it would be nice, if the last edited package could stay highlighted after being sorted in, so that the user immediately sees the results of his time editing.
also, an option to sort by mission type would be useful.
